src/test/java/hu/rxd/toolbox/qtest/diff/DiffClassificatorTest.java

```diff
@@ -24,7 +24,7 @@
   private String fileName;
   private String expected;
 
-  @Parameters(name = "{0}")
+  @Parameters
```

Comment:
IIRC without name = "{0}" the ide will not show the name of the testcase; and it will use numbers like 0,1,2...